tools/serverpod_cli/lib/src/create/create.dart

```diff
@@ -378,6 +377,10 @@ void _copyServerUpgrade(
           slotName: 'SERVICE_SECRET_DEVELOPMENT',
           replacement: generateRandomString(),
         ),
+        Replacement(
+          slotName: 'SERVICE_SECRET_TEST',
+          replacement: generateRandomString(),
+        ),
```

Comment:
Comment: It is a shame we place the namespace (`TEST`) keyword at the end here but after `DB` in the db case... :(

Reply:
But this is nothing we can fix right now.

Reply:
Yea 😿